app/registries/foreman/plugin/rbac_support.rb

```diff
@@ -34,7 +37,7 @@ def add_permissions_to_default_roles(roles)
 
       def check_role_name_before_extending(role_name)
         msg = "Invalid role name, only 'Viewer' or 'Manager' is allowed to be extended from a plugin"
-        raise Foreman::Exception.new msg if role_name != "Viewer" && role_name != "Manager"
+        raise Foreman::Exception.new msg if role_name != "Viewer" && role_name != "Manager" && role_name != "Organization admin"
```

Comment:
Looks like it's time to add a constant array `SHIPPED_ROLES` and check whether `role_name` is in there